invenio_deposit/ext.py

```diff
@@ -132,8 +132,8 @@ def init_app(self, app):
             app.config['DEPOSIT_REST_ENDPOINTS']
         )
 
-        @blueprint.record_once
-        def extend_default_endpoint_prefixes(_):
+        @app.before_first_request
```

Comment:
Will you add a``# FIXME`` here, with a note explaining what we talked IRL Friday (regarding request context and triggering before first request in order to make URL generation work for deposits in e.g. Celery tasks).